Example/Example/ViewController.swift

```diff
@@ -21,16 +21,33 @@ class ViewController: FormViewController {
                 $0.countryPlaceholder = "Country"
                 $0.postalCodePlaceholder = "Zip code"
         }
-            +++ Section()
-                <<< MyPostalAddressRow() {
-                    $0.streetPlaceholder = "Street"
-                    $0.cityPlaceholder = "City"
-                    $0.postalCodePlaceholder = "Zip code"
-                }.cellSetup({ (cell, row) in
-                    cell.streetTextField?.font = .systemFont(ofSize: 18)
-                    cell.postalCodeTextField?.font = .systemFont(ofSize: 18)
-                    cell.cityTextField?.font = .systemFont(ofSize: 18)
-                })
+			
+		+++ Section()
+			<<< PostalAddressRow() {
+				$0.streetPlaceholder = "Street"
+				$0.statePlaceholder = "State"
+				$0.cityPlaceholder = "City"
+				$0.countryPlaceholder = "Country"
+				$0.postalCodePlaceholder = "Zip code"
+				
+				$0.countrySelectorRow = PushRow<String>(){
+					$0.options = ["GB","US"]
+					$0.displayValueFor = { guard let isoCode = $0 else{ return nil }
+						return Locale.current.localizedString(forRegionCode: isoCode)
+					}
+				}
+			}
+			
+		+++ Section()
+			<<< MyPostalAddressRow() {
+                $0.streetPlaceholder = "Street"
```

Comment:
Could you align this line to its context? Maybe if you set Xcode to replace tabs with spaces